sys-apps/xdg-desktop-portal/xdg-desktop-portal-1.18.0.ebuild

```diff
@@ -0,0 +1,71 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson systemd
```

Comment:
used python-r1 instead because of
pkgcheck scan
```
InvalidDepend: version 1.18.0: failed parsing DEPEND: '>=dev-libs/glib-2.66:2 dev-libs/json-glib >=sys-fs/fuse-3.10.0:3[suid] x11-libs/gdk-pixbuf geolocation? ( >=app-misc/geoclue-2.5.3:2.0 ) >=media-video/pipewire-0.3:= systemd? ( sys-apps/systemd ) test? ( || ( dev-lang/python:3.12 dev-lang/python:3.11 dev-lang/python:3.10 ) dev-python/pytest[] dev-libs/libportal ) man? ( dev-python/docutils )' is unparseable: flagged token- invalid package atom: 'dev-python/pytest[]': empty use dep detected
```